py/torch_tensorrt/dynamo/conversion/impl/select.py

```diff
@@ -390,3 +391,52 @@ def index_select(
     set_layer_name(gather_layer, target, f"{name}_gather", source_ir)
 
     return gather_layer.get_output(0)
+
+
+def scatter(
+    ctx: ConversionContext,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input: TRTTensor,
+    dim: int,
+    index: Union[TRTTensor, np.ndarray, torch.Tensor],
+    src: Union[TRTTensor, int, float],
+) -> TRTTensor:
+    input_shape = input.shape
+    index_shape = index.shape
+    index_shape_list = list(index.shape)
+    if not (isinstance(index, TRTTensor)):
+        if isinstance(index, torch.Tensor):
+            if index.dtype == torch.int64:
+                index = index.to(torch.int32)
+        elif isinstance(index, np.ndarray):
+            if index.dtype == np.int64:
+                index = index.astype(np.int32)
+        index = get_trt_tensor(ctx, index, f"_index_tensor")
+    dim = get_positive_dim(dim, len(input_shape))
+    dynamic_shape = has_dynamic_shape(input.shape)
+    if dynamic_shape:
+        # Check whether slice target dim is dynamic shape dim
+        assert input.shape[dim] != -1, "Can't scatter on negative shape dimension!"
```

Comment:
This can be removed, and should be addressed by #2796

Reply:
So after this PR is merged the above would need to be addressed accordingly?

Reply:
Yes, I believe the above would have to be addressed in a future PR which adds dynamic shape support for this converter. With #2796, this converter would be registered as static-only by default